public/pages/DetectorResults/containers/AnomalyHistory.tsx

```diff
@@ -144,10 +160,8 @@ export const AnomalyHistory = (props: AnomalyHistoryProps) => {
     1
   );
 
-  const fetchRawAnomalyResults = async (
-    shouldSetAtomicAnomalyResults: boolean
-  ) => {
-    if (shouldSetAtomicAnomalyResults) {
+  const fetchRawAnomalyResults = async (showLoading: boolean) => {
```

Comment:
As we discussed, get raw anomaly result may have performance issue if user choose a long period for a HC detector which has thousands of entities. Suppose we get 1K result for 1 entity, 1K entities will have 1M results.
Add some comments & TODO here and fix it later.

Reply:
Delete this, comment wrong method